packages/asgardex-bitcoin/src/client.ts

```diff
@@ -34,7 +44,7 @@ class Client implements BitcoinClient, AsgardexClient {
   nodeApiKey = ''
 
   // Client is initialised with network type
-  constructor(_net: Network = Network.TEST, _nodeUrl = '', _nodeApiKey = '', _phrase?: string) {
+  constructor(_net: Network = 'testnet', _nodeUrl = '', _nodeApiKey = '', _phrase?: string) {
```

Comment:
Pls keep in mind constructor will take a parameter object [as described in latest specs](https://gitlab.com/thorchain/asgardex-common/asgardex-client/-/blob/35a3e0382a12692a6af95d10b9c64682f1158400/README.md#config-and-setup) - something like that:

```
type ClientParams = AsgardexClientParams & {
  nodeApiKey: string
  nodeUrl: string
  ...
}

class Client implements AsgardexClient, BitcoinClient {

    constructor(params: ClientParams) {
      ...
    }
}
```